collectfast/boto.py

```diff
@@ -1,26 +0,0 @@
-from . import settings
-
-try:
-    from storages.backends.s3boto import S3BotoStorage
-    from storages.backends.s3boto3 import S3Boto3Storage
-    has_boto = True
-except:
-    has_boto = False
-
-
-def is_boto3(storage):
-    return has_boto and isinstance(storage, S3Boto3Storage)
-
-
-def is_boto(storage):
-    return has_boto and (
-        isinstance(storage, S3BotoStorage) or
-        isinstance(storage, S3Boto3Storage))
-
-
-def reset_connection(storage):
```

Comment:
Since this logic is only specific to Boto3, I moved it to `S3Boto3StorageExtensions` so this file could be removed